samples/javascript/videoshare/proto/pubsub.proto

```diff
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package video;
+
+import "google/protobuf/any.proto";
+
+message UpstreamMessage {
+    oneof message {
+        SendToGroupMessage send_to_group_message = 1;
+        EventMessage event_message = 5;
```

Comment:
how about use 1,2,3,4.... in sequence

Reply:
This can't change, it must keep consistent. Why we reserve some number because we think we will add some send related message type